dashboards/linux.summary.json

```diff
@@ -3,14 +3,14 @@
     "id" : null,
     "name" : "Linux Summary",
     "description" : null,
-    "layout" : "{\"version\":\"2\",\"contents\":[{\"widgets\":[\"ac009d5f-97af-43aa-b6b5-acef5e0749db\",\"9199dfc5-52ce-49bc-b839-1e10dd854822\",\"a5b64a22-0ec9-423d-9010-9995e2146a3d\"],\"autoGenerated\":false},{\"widgets\":[\"9e030328-626a-4d1c-9436-6f14a38d2c15\",\"c4d35102-159d-4c10-ab79-11b842a15696\",\"d2a6970c-e0a7-4850-b7b7-a5a67d8f7f90\"],\"autoGenerated\":false},{\"widgets\":[\"37b7ab07-e34a-43c3-8a70-7742579bcc9b\",\"4764b195-9d48-4f0f-a7ba-47c74167b88e\"],\"autoGenerated\":false}]}",
+    "layout" : "{\"version\":\"2\",\"contents\":[{\"widgets\":[\"bf72bc89-3235-4c3b-8c21-a7bb992f5f12\",\"9199dfc5-52ce-49bc-b839-1e10dd854822\"],\"autoGenerated\":false},{\"widgets\":[\"9e030328-626a-4d1c-9436-6f14a38d2c15\",\"c4d35102-159d-4c10-ab79-11b842a15696\",\"d2a6970c-e0a7-4850-b7b7-a5a67d8f7f90\"],\"autoGenerated\":false},{\"widgets\":[\"37b7ab07-e34a-43c3-8a70-7742579bcc9b\",\"4764b195-9d48-4f0f-a7ba-47c74167b88e\"],\"autoGenerated\":false}]}",
```

Comment:
@rclagett this layout should be null, like https://github.com/netuitive-community-packages/netuitive-packages-linux/blob/dev/dashboards/linux.element.detail.json